tests/test_server/mod.rs

```diff
@@ -0,0 +1,29 @@
+
+use std::process::Child;
+
+pub struct ExampleServer(Child);
+
+impl std::ops::Drop for ExampleServer {
+    fn drop(&mut self) {
+        let Ok(_) = self.0.kill()else{
+            println!("faild kill a process. ");
+            return;
+        };
+    }
+}
+
+pub fn run_example_server(example_name: &str, port_number: usize) -> ExampleServer {
+    let temp = assert_fs::TempDir::new().unwrap();
+    ExampleServer(
+        escargot::CargoBuild::new()
+            .example(example_name)
+            .manifest_path("Cargo.toml")
+            .target_dir(temp.path())
```

Comment:
this is not needed, right? As far as I can see from the documentation it is not a required method for this builder to succeed, and I assume that it will just use your regular target dir, which is ok?